unicorn/templates/default/nginx_unicorn_web_app.erb

```diff
@@ -48,9 +48,15 @@ server {
     proxy_redirect off;
 
     <% if @application[:force_ssl] %>
+    set $https_forward "https://$host$uri";
+    # Don't disclose private IP addresses when no host specified (HTTP/1.0)
+    if ($host ~ "\d+\.\d+\.\d+\.\d+") {
+      set $https_forward "https://www.<%= @application[:domains].first %>.com$uri";
```

Comment:
Sort of. One of the default domains is the app name, which is why this is working on the topic stacks. We should probably just use `@application[:domains].first` as we do elsewhere. (The app name is always second last. With the last being the server's hostname.) Otherwise we'll end up with `https://www.www.domain.com.com` in production which would not be ideal.